packages/common/src/types/gethGenesis.ts

```diff
@@ -0,0 +1,84 @@
+/**
+ * Interface for Geth Genesis Config
+ */
+export interface GethGenesisConfig {
+  chainId: number
+  homesteadBlock?: number
+  daoForkBlock?: number
+  daoForkSupport?: boolean
+  eip150Block?: number
+  eip150Hash?: string
+  eip155Block?: number
+  eip158Block?: number
+  byzantiumBlock?: number
+  constantinopleBlock?: number
+  petersburgBlock?: number
+  istanbulBlock?: number
+  muirGlacierBlock?: number
+  berlinBlock?: number
+  londonBlock?: number
+  mergeForkBlock?: number
+  cancunBlock?: number
+  shanghaiTime?: number
+  cancunTime?: number
+  terminalTotalDifficulty?: number
+  terminalTotalDifficultyPassed?: boolean
+  ethash?: {}
```

Comment:
I think `blobSchedule` should be added here, like in Hoodi https://github.com/eth-clients/hoodi/blob/7f71638afc431675c19b9e306998b9d4925b7cec/metadata/genesis.json#L19

Reply:
Thanks for the note, just added it